app/core/core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/material3/HedvigMaterial3Theme.kt

```diff
@@ -104,7 +104,9 @@ private fun darkColorScheme(hedvigTonalPalette: HedvigTonalPalette) = darkColorS
 
 internal fun darkHedvigColorScheme(hedvigTonalPalette: HedvigTonalPalette) = HedvigMaterial3ColorScheme(
   containedButtonContainer = hedvigTonalPalette.greyscale25,
-  onContainedButtonContainer = hedvigTonalPalette.greyscale1000,
+  onContainedButtonContainer = hedvigTonalPalette.greyscale100,
+  secondaryContainedButtonContainer = hedvigTonalPalette.greyscale700,
```

Comment:
Was this because 700 looked more close to design compared to 800?
If yes, can we double check with iOS? Because I got 800 from them when I asked for this.